test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -293,7 +293,7 @@ foreach(test_name IN LISTS NNPA_TEST_LIST)
   set(ENV_TEST_CASE_BY_USER "${ENV_TEST_CASE_BY_USER} ${test_name}")
 endforeach()
 
-set(NNPA_TESTS_ENVS TEST_MACCEL=NNPA TEST_CASE_BY_USER=${ENV_TEST_CASE_BY_USER} TEST_ATOL=0.01 TEST_RTOL=0.05)
+set(NNPA_TESTS_ENVS TEST_MACCEL=NNPA TEST_CASE_BY_USER=${ENV_TEST_CASE_BY_USER} TEST_ATOL=0.01 TEST_RTOL=0.05) # TEST_ULPTOL=DFLOAT:2
```

Comment:
Can you comment on why the `TEST_ULPTOL` is not used?

Reply:
@AlexandreEichenberger Thank you for the comment.
I investigated the errors for the backend test cases. The following graph shows relationship between output values of backend test on NNPA and their errors. In the graph each point shows each output of the backend test on NNPA.

![image](https://user-images.githubusercontent.com/18544851/180676162-3a5d4c1c-d6f3-4da1-9ce0-7bfdb4e250ce.png)

I realized that ULP-based tolerance setting  is useful only when the output values are larger than 10 and most of the current backend test case output values are less than 10.
So that I am thinking to close this PR.  I will investigate other test cases and will use this code if it is useful.
Your comments are very welcome.

Reply:
Nice graph, really explain the situation well. I think it continue to show that "managing" the values (as you said, output smaller than 10) is really a feature of the model and its training weight. Long term solution for actual models remain well trained weights for the available precision of the NNPA